tests/dummy/app/routes/docs/helper.js

```diff
@@ -0,0 +1,30 @@
+/* global require */
+
+import Ember from 'ember';
+
+const helperDescs = {
```

Comment:
I think I have a better idea in mind.
What if we use the `{{component` helper like:

```hbs
{{component helperName}}
```

Then we can still write the docs on templates, where they should live.

My plan is to extend this to the other components, eventually, but I'd like to take this opportunity to do it on helpers.

Can you please arrange this? Let me know if you need any help. Here or on slack.